pkg/gadgets/internal/socketenricher/tracer_test.go

```diff
@@ -203,32 +204,35 @@ func TestSocketEnricherBind(t *testing.T) {
 			if test.validateEvent != nil {
 				test.validateEvent(t, runner.Info, port, entries)
 			}
+
+			if fd != -1 {
+				unix.Close(fd)
+			}
```

Comment:
What about using `t.Cleanup()` to run it on L189. Otherwise it won't run if the test fails leaking the fd.

Reply:
Updated.